test/bin/scenario.sh

```diff
@@ -16,11 +16,22 @@ PULL_SECRET_CONTENT="$(jq -c . "${PULL_SECRET}")"
 PUBLIC_IP=${PUBLIC_IP:-""}  # may be overridden in global settings file
 VM_BOOT_TIMEOUT=8m
 
+trap "sos_report" EXIT
```

Comment:
We only want the trap installed when we're running the test. Otherwise it will try to run on cleanup when we're deleting the VM.